### PR TITLE
AAP-5413: Created AAP 2.1.3 release notes

### PR DESCRIPTION
**Pull request by sayjadha**
This PR addresses JIRA issue [AAP-5413](https://issues.redhat.com/browse/AAP-5413).

Following are the changes:

- Added platform and hub release notes for AAP 2.1.3

Author: sayjadha

### CODE REVIEW

File: release-notes/master.adoc
Comment (jbradberry) [inline release-notes/master.adoc:24]: Am I mistaken in thinking that `topics/aap-212.adoc` is going to need to be in here as well?  Or are we planning on waiting on the merge of that PR and then resolving the conflict?

Comment (sayjadha) [reply release-notes/master.adoc:24]: Hi @jbradberry : You're right. The AAP 2.1.2 release notes are being tracked in PR #[552](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/pull/552), and will be available once the review is complete and PR is merged.

**Review by tvo318 — CHANGES_REQUESTED** [review]

File: release-notes/topics/aap-213.adoc
Comment (ariordan-redhat) [inline release-notes/topics/aap-213.adoc:25]:
```suggestion
* Installation of Red Hat Single Sign On (SSO) on private automation hub no longer fails
```

File: release-notes/topics/aap-213.adoc
Comment (ariordan-redhat) [inline release-notes/topics/aap-213.adoc:26]:
```suggestion
* Bundle installation of private automationhHub no longer pushes images with missing `ee_image_base` that leads to installation failure
```

File: release-notes/topics/aap-213.adoc
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc]: Made the change.

File: release-notes/topics/aap-213.adoc
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc]: Made the change.

File: release-notes/topics/aap-213.adoc
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:19]: Made the edit.

File: release-notes/topics/aap-213.adoc
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:22]: Made the change.

File: release-notes/topics/aap-213.adoc
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:25]: Made the change.

File: release-notes/topics/aap-213.adoc
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:26]: Made the change.

File: release-notes/topics/aap-213.adoc
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:34]: Automation controller UI, added the info.

**Comment by sayjadha** [comment]
Hi @tvo318, @ariordan-redhat: Thank you for the review. Your feedback is incorporated.

**Review by ariordan-redhat — APPROVED** [review]

**Pull request merged**
